app/views/posts/_post.json.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.extract! post, :id, :name, :title, :content, :description, :visibility, :published_date, :status, :importance, :rating, :created_at, :updated_at
```

Comment:
jbuilder is old-fashoined, prefferable to  use ActiveModel Serializer if you have time